Comparison Engine/Comparison Engine/GoogleMap/MapController.cs

```diff
@@ -24,6 +24,10 @@ private MapController()
         {
         }
 
+        // set default values so it would compile, but will need to set these depending on user or from current location
```

Comment:
already possible to make some basic methods to set these values, since I doubt we're not changing them directly

Reply:
same trello task